lib/hw_unique_key/Kconfig

```diff
@@ -63,3 +63,14 @@ config HW_UNIQUE_KEY_PARTITION_SIZE
 	help
 	  Size of the flash page for storing the HUK.
 	  Only needed when there is no KMU for storing keys (nRF52840).
+
+config READ_HW_UNIQUE_KEY
```

Comment:
I think that it is better to be consistent here. So I would propose to rename the symbol READ_HW_UNIQUE_KEY -> HW_UNIQUE_KEY_READ

Reply:
+1.

It is convention to start with the module, and then suffix with features of the module.

Reply:
Changing it